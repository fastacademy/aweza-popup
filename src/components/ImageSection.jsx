import { h, Component } from 'preact'

export default class ImageSection extends Component {
  render () {
    return (
      <div class="aweza-image">
        <div class="aweza-section-title">
          <b>Image</b>
        </div>
        <img src={this.props.image}/>
      </div>
    )
  }
}