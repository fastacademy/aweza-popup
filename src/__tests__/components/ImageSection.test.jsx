// noinspection ES6UnusedImports
import { h } from 'preact';
import render from 'preact-render-to-json';
import ImageSection from "../../components/ImageSection";
import {terms} from "../dummydata";

test('ImageSection component', () => {
    const tree = render(<ImageSection image={terms[0].image}/>);

    expect(tree).toMatchSnapshot();
});
