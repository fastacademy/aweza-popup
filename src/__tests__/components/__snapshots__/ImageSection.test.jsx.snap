// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageSection component 1`] = `
<div
  class="aweza-image"
>
  <div
    class="aweza-section-title"
  >
    <b>
      Image
    </b>
  </div>
  <img
    src="http://platform.fastacademy.co.za/media/view/2017/06/parabola_21.png"
  />
</div>
`;
